secrets.tf

```diff
@@ -107,3 +107,7 @@ variable "star_tasks_build_tls_key" {
 variable "taskcluster_net_san_tls_key" {
   type = "string"
 }
+
+variable "relengapi_token" {
```

Comment:
This should be named `tc_worker_docker_relengapi_token` since the env variable in secrets is `TF_VAR_tc_worker_docker_relengapi_token` and the variable referenced below is `tc_worker_docker_relengapi_token`